src/modules/trash/components/DestroyConfirm.tsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react'
+
+import { splitFilename } from 'cozy-client/dist/models/file'
+import { ConfirmDialog } from 'cozy-ui/transpiled/react/CozyDialogs'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+import ForbiddenIcon from 'cozy-ui/transpiled/react/Icons/Forbidden'
+import RestoreIcon from 'cozy-ui/transpiled/react/Icons/Restore'
+import List from 'cozy-ui/transpiled/react/List'
+import ListItem from 'cozy-ui/transpiled/react/ListItem'
+import ListItemIcon from 'cozy-ui/transpiled/react/ListItemIcon'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import Button from 'cozy-ui/transpiled/react/deprecated/Button'
+import { useI18n } from 'cozy-ui/transpiled/react/providers/I18n'
+
+import { File } from 'components/FolderPicker/types'
+import { getEntriesTypeTranslated } from 'lib/entries'
+
+interface DestroyConfirmProps {
+  files: File[]
+  onClose: () => void
+  onConfirm: () => Promise<void>
+}
+
+const DestroyConfirm: React.FC<DestroyConfirmProps> = ({
+  files,
+  onClose,
+  onConfirm
+}) => {
+  const { t } = useI18n()
+  const [isBusy, setBusy] = useState(false)
+
+  const handleDestroy = async (): Promise<void> => {
+    try {
+      setBusy(true)
+      await onConfirm()
+    } finally {
```

Comment:
we don't handle the error scenario with some UI feedback here?

Reply:
Good point, I've added alerts in case of success or error